packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -95,24 +97,22 @@ export class MonacoEditorLanguageClientWrapper {
     private languageClient: MonacoLanguageClient | undefined;
     private worker: Worker | undefined;
 
-    private editor: EditorClassic | EditorVscodeApi | undefined;
+    private editorApp: EditorAppClassic | EditorAppVscodeApi | undefined;
 
     private id: string;
     private htmlElement: HTMLElement;
-    private useVscodeConfig: boolean;
     private serviceConfig: InitializeServiceConfig;
     private languageClientConfig: LanguageClientConfig;
 
     private init(userConfig: UserConfig) {
-        if (userConfig.editorConfig.useDiffEditor) {
-            if (!userConfig.editorConfig.codeOriginal) {
+        if (userConfig.editorContentConfig.useDiffEditor) {
```

Comment:
Can combine both conditions (this one and the one below) into a single branch, if there's just the one error that comes out when both are met, and nothing happens after the first condition alone.

Reply:
Of course 🙂

Reply:
Fixed ⬇️